### DISCUSSION TITLE
my candidates for luarocks-tag-release

### DISCUSSION THREAD

**Discussion by teto** [nvim-neorocks/luarocks-tag-release#68] [General]
Here are a few plugins I would like to "luarocks-tag-release" because they are meant to be dependencies:
- https://github.com/MunifTanjim/nui.nvim/
- https://github.com/romgrk/kui.nvim
- https://github.com/edluffy/hologram.nvim#usage
- some treesitter grammars  like https://github.com/rest-nvim/tree-sitter-http (I got merging rights over there)

And some fun applications 
- https://github.com/romgrk/kirby.nvim
- https://github.com/kndndrj/nvim-dbee
- https://github.com/giusgad/pets.nvim

I would usually list them on the nixpkgs tracker but this one has fewer issues, so it's easier to track the issue

**Comment by teto** [comment]
haha didn't know you could do this, yeah conversations are better for this. I also saw this one I dont use but that is using dependencies (the primary usecase for our luarocks approach): 
https://github.com/nvim-neo-tree/neo-tree.nvim

**Reply by mrcjkb** [reply]
I hate tree-style file managers. But yeah, it's a good use case.

**Comment by mrcjkb** [comment]
nui.nvim already publishes to luarocks, just not using our workflow 😄 

https://github.com/MunifTanjim/nui.nvim/blob/main/.github/workflows/publish.yml
